bin/node/runtime/src/lib.rs

```diff
@@ -677,9 +678,7 @@ impl onchain::Config for OnChainSeqPhragmen {
 	>;
 	type DataProvider = <Runtime as pallet_election_provider_multi_phase::Config>::DataProvider;
 	type WeightInfo = frame_election_provider_support::weights::SubstrateWeight<Runtime>;
-}
-
-impl onchain::BoundedConfig for OnChainSeqPhragmen {
+	type MaxWinners = <Runtime as pallet_election_provider_multi_phase::Config>::MaxWinners;
 	type VotersBound = MaxElectingVoters;
 	type TargetsBound = ConstU32<2_000>;
```

Comment:
These values are a bit of a bad example: a fallback should have a smaller input. Something that says "these bounds are for oncahin fallback and should be small".

```suggestion
	type VotersBound = MaxElectingVoters / 8;
	type TargetsBound = MaxWinners / 8;
```